core/src/main/java/org/geysermc/geyser/text/MinecraftLocale.java

```diff
@@ -168,24 +196,12 @@ private static boolean loadLocale(String locale) {
                 Map.Entry<String, JsonNode> entry = localeIterator.next();
                 langMap.put(entry.getKey(), entry.getValue().asText());
             }
-
-            String bedrockLocale = locale.toLowerCase(Locale.ROOT);
-            if (bedrockLocale.equals("no_no")) {
-                // Store this locale under the Bedrock locale so we don't need to do this check over and over
-                bedrockLocale = "nb_no";
-            }
-
-            // Insert the locale into the mappings
-            LOCALE_MAPPINGS.put(bedrockLocale, langMap);
-
-            try {
-                localeStream.close();
-            } catch (IOException e) {
-                throw new AssertionError(GeyserLocale.getLocaleStringLog("geyser.locale.fail.file", locale, e.getMessage()));
-            }
-            return true;
-        } else {
-            return false;
+            localeStream.close();
+            return langMap;
+        } catch (FileNotFoundException e){
+            throw new AssertionError(GeyserLocale.getLocaleStringLog("geyser.locale.fail.file", locale, e.getMessage()));
+        } catch (Exception e) {
+            throw new AssertionError(GeyserLocale.getLocaleStringLog("geyser.locale.fail.json", locale), e);
```

Comment:
Do we really want to hard fail here?
I see that it used to be this way though, so, whatever

Reply:
That's unchanged from before